@here/harp-mapview/lib/MapViewFog.ts

```diff
@@ -6,14 +6,14 @@
 
 import { Theme } from "@here/harp-datasource-protocol";
 import { HighPrecisionLineMaterial, SolidLineMaterial } from "@here/harp-materials";
-import { Fog, Material, Mesh } from "three";
+import { Fog, Material, Mesh, OrthographicCamera, PerspectiveCamera, Scene } from "three";
 
 /**
  * Manages the fog display in [[MapView]].
  */
 export class MapViewFog {
     private m_enabled: boolean = true;
-    private m_fog: THREE.Fog = new Fog(0x000000); // Default color asked by DefinitelyTyped.
+    private m_fog: Fog = new Fog(0x000000); // Default color asked by DefinitelyTyped.
```

Comment:
Now that we can rely on ThreeJS, this may be removed

Reply:
@Astrak:
I don't follow.
What exactly can be removed ?

Reply:
The default color. @types/three required a colour to initialize the fog, but THREE.Fog actually does not. Depends on the implementation in the ThreeJS'typings.

Reply:
Actually, Fog requires color ...
https://github.com/mrdoob/three.js/blob/dev/src/scenes/Fog.d.ts#L14

Reply:
Interesting. I wonder if they have just copied the types from @types/three then. Because the argument is only used to create a `Color` object, where the argument is optional in the constructor.